pom.xml

```diff
@@ -22,108 +22,44 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Git+Plugin</url>
 
   <properties>
-    <findbugs-maven-plugin.version>3.0.3</findbugs-maven-plugin.version>
-    <jacoco-maven-plugin.version>0.7.4.201502262128</jacoco-maven-plugin.version>
-    <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
-    <maven-enforcer-plugin.version>1.4.1</maven-enforcer-plugin.version>
-    <maven-hpi-plugin.version>1.117</maven-hpi-plugin.version>
-    <maven-localizer-plugin.version>1.23</maven-localizer-plugin.version>
-    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
-    <maven-site-plugin.version>3.5</maven-site-plugin.version>
-    <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <jenkins.version>1.609</jenkins.version>
```

Comment:
I've been considering using the current 2.5.0-beta branch as the place to make the switch from Java 6 to Java 7.  Is it allowed to switch from 1.609 to 1.625 (the first version that requires Java 7)?